### DISCUSSION TITLE
Unable to download wp core. Retrying...

### DISCUSSION THREAD

**Discussion by oitent** [EasyEngine/easyengine#1873] [Support Requests] [answered]
As of last Wordpress Update 6.7 I simply cannot create a new wordpress site.

Im using EE4 in a brand new DigitalOcean droplet. 

Command used (with example placeholders):
```
ee site create mysitedomain.com.br --type=wp --title="My Title" --locale=pt_BR --ssl=le
```
Anyone with the same issue?

```

Starting site creation.
Configuring project.
Creating WordPress site alfecafe.com.br
Copying configuration files.
Starting site's services.
Downloading and configuring WordPress.
Unable to download wp core. Retrying...
Unable to download wp core. Retrying...
Unable to download wp core. Retrying...
Unable to download wp core. Retrying...
Unable to download wp core. Retrying...
Warning: Couldn't connect to global-db:3306 or there was issue in `wp config create`. Please check logs.
Warning: Initiating clean-up.
```

**Comment by L0RD-ZER0** [comment]
Thank you for notifying us about this!

Our team looked into it, and it seems to be stemming from a PHP Out of Memory issue during the time of site creation. Our team is actively working on a fix and we will get it fixed in the next release asap.

Thanks for your patience and continued support.

**Comment by mrrobot47** [comment, marked as answer]
It has been fixed in new release [EE v4.7.6](https://github.com/EasyEngine/easyengine/releases/tag/v4.7.6). Please update to latest EE via `ee cli update`

Issue ref: https://github.com/EasyEngine/easyengine/issues/1874

**Comment by oitent** [comment]
Success guys, thank you so much!
<img width="571" alt="image" src="https://github.com/user-attachments/assets/bef21823-52e4-4dcd-8273-73258304e157">
